packages/cloudflare-optimizer-scripts/src/index.js

```diff
@@ -85,7 +94,17 @@ async function handleRequest(event, config) {
     }
   }
 
-  const response = await fetch(request);
+  const response = await fetch(request, {redirect: 'manual'});
+
+  // Redirect based statuses should be returned
+  if (isRedirect(response)) {
```

Comment:
@samouri Would this suffice or should I move this logic into rewriteRedirectHeader and rename it to `maybeRewriteRedirectHeader`  . In that case, would it be okay if based on the return value of the function we are either returning the response or continuing with the below lines of code? 

Just wanted to confirm the pattern you'll prefer.

Reply:
Either way is fine with me! But consistency would be good.

Right now one of them is written as nested if conditions whereas the other combines the conditions with `&&` which is preferred in this case.